src/strings/PowerFxResources.en-US.resx

```diff
@@ -4846,4 +4846,11 @@
     <value>The 'User-defined types' experimental feature is not enabled.</value>
     <comment>Error message returned when user uses User-defined types with the User-defined types feature flag turned off.</comment>
   </data>
+  <data name="ErrUserDefinedTypeIncorrectSyntax" xml:space="preserve">
+    <value>Incorrect syntax `=` used to define User-defined type.</value>
```

Comment:
>value [](http://example.com/codeflow?start=5&length=5)

If we know the problem and how it can be fixed, we should  tell the customer that, something like "Incorrect syntax: '=' cannot be used to declare user-defined types. Use ':=' instead."